src/module_compilation.ml

```diff
@@ -196,14 +196,6 @@ let build_module ~dep_graphs cctx m =
     SC.add_rules sctx ~dir
       (Js_of_ocaml_rules.build_cm cctx ~js_of_ocaml ~src ~target))
 
-let build_modules ~dep_graphs cctx =
-  Module.Name.Map.iter
-    (match CC.alias_module cctx with
-     | None -> CC.modules cctx
-     | Some (m : Module.t) ->
-       Module.Name.Map.remove (CC.modules cctx) (Module.name m))
-    ~f:(build_module cctx ~dep_graphs)
```

Comment:
I can't see where the "remove alias module" logic is moved to, is it not necesary anymore?

Reply:
Yeah, it's basically not necessary.

* When we build the user written modules, we pass them explicitly
* When was pass the alias module, we just pass it explicitly.

Reply:
alright